### DISCUSSION TITLE
Options for TLS_FLAVOR in mailu.env

### DISCUSSION THREAD

**Discussion by FaBMak** [Mailu/Mailu#3660] [User support]
The mailu.env only shows the different options for TLS_FLAVOR, but I couldn't found what the differences between them.

`# Choose how secure connections will behave (value: letsencrypt, cert, notls, mail, mail-letsencrypt)`

So, here's some questions? 

- letsencrypt - does letsencrypt obtain certs for both, front and mail ?
-  mail - does mail obtain certs only for mail ?
- mail-letsencrypt - what's the difference between mail and mail-letsencrypt ?

Can someone point me to the right documentation?

**Comment by ghostwheel42** [comment]
https://mailu.io/2.0/compose/setup.html#tls-certificates

**Reply by gbanach** [reply]
I suppose the question will be valid as long as: https://setup.mailu.io/2024.06/ will provide information from this page: https://mailu.io/2024.06/compose/setup.html#tls-certificates

**Comment by ghostwheel42** [comment]
In 2024.06 the mail-* flavors have been removed and supported flavors are cert, letsencrypt and notls
But I see comments have not been cleaned.
